{"name": "test_wrong_password", "status": "passed", "description": "\"test_wrong_password\" is the test that try auth with wrong password", "steps": [{"name": "authorization wrong password", "status": "passed", "start": 1694881195686, "stop": 1694881203146}], "attachments": [{"name": "stdout", "source": "ea465234-5e73-4e94-9195-b8f863038128-attachment.txt", "type": "text/plain"}], "start": 1694881191697, "stop": 1694881203269, "uuid": "7ad7abd7-1598-4b1b-93c4-bf265a5464fe", "historyId": "b0cebe0be7821dec708730b8b3e41e3e", "testCaseId": "b0cebe0be7821dec708730b8b3e41e3e", "fullName": "tests.test_authorizations#test_wrong_password", "labels": [{"name": "tag", "value": "order(2)"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_authorizations"}, {"name": "host", "value": "MacBook-Air-Aleksej.local"}, {"name": "thread", "value": "20432-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_authorizations"}]}